doc/content/htgr/generic-pbr/generic-pbr_results.md

```diff
@@ -0,0 +1,311 @@
+relatively constant with each experiencing minor changes. As a result, the
+respective reactivities remain largely constant and produce a total reactivity
+of approximately zero. Lastly, during the ramp up stage, the fuel and kernel
+temperatures increase which then cause the reactivity of the fuel to decrease.
+Conversely, the increased coolant flow reduces the moderator temperature and leads
+to an increase of moderator reactivity. The reflector temperature also decreases
+in this stage, resulting in a decrease of reflector reactivity. Overall, the total
+reactivity shows a sharp initial increase before dropping gradually as the flow rate
+is increased back to the nominal level. During the course of the transient, the
+total reactivity is shown to be primarily influenced by the reactivity feedback
+of the moderator.  
+
+The mean fuel and kernel temperatures, moderator temperature, and reflector
+temperature predicted by SAM are shown in [Tfuel_avg], [Tmoderator_avg], and
+[Treflector_avg], respectively. Note that the SAM results shown here are the
+volume-averaged temperature of a particular component. Volume-averaging is
+necessary to ensure that the mean temperatures are not skewed by the number of
+sections of a component. For instance, due to its complex geometry, the lower
+reflector region is comprised of a greater number of sections compared to the
+side reflector which has a comparatively simpler geometry. Thus, without
+volume-averaging, the mean temperature will be skewed towards the lower reflector
+region.
+
+For the SAM prediction, during the initial ramp down, despite the increasing
+fuel temperature, the kernel temperature decreases due to the reduction of
+reactor power as shown in [reactor_power]. As the fuel (Doppler) reactivity
+feedback is calculated with the kernel temperature, a decrease in the kernel
+temperature causes the fuel reactivity to increase. At the same time, the
+moderator temperature increases and leads to a decrease of reactivity. On the
+other hand, given that the reflector reactivity coefficients are primarily
+positive, an increase in the reflector temperature causes the reflector reactivity
+to increase. It is also observed that the total reactivity is dominated by
+the moderator reactivity. During the constant flow rate stage, the kernel,
+moderator, and reflector temperatures are relatively unchanged with each
+experiencing minor changes. As a result, the respective reactivities remain
+largely constant and produce a total reactivity of approximately zero.
+
+Lastly, during the ramp up stage, the fuel and kernel temperatures increase and
+cause the reactivity of the fuel to decrease. Conversely, the increased coolant
+flow reduces the moderator temperature and leads to an increase of moderator
+reactivity. The reflector temperature also decreases in this stage, resulting
+in a decrease of reflector reactivity. The resultant total reactivity shows a
```

Comment:
why is colder reflector worse for reactivity? curious

Reply:
The reflectors have positive reactivity coefficients so a decrease in reflector will cause the reflector reactivity to decrease as well.